devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy

```diff
@@ -209,17 +222,138 @@ def ping() {
+		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 101, size: 4, scaledConfigurationValue: 4)))   // report power in watts
+		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 111, size: 4, scaledConfigurationValue: 300)))  // every 5 min
+		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 102, size: 4, scaledConfigurationValue: 8)))   // report energy in kWh
+		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 112, size: 4, scaledConfigurationValue: 300)))  // every 5 min
+		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 103, size: 4, scaledConfigurationValue: 0)))   // no third report
+		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 113, size: 4, scaledConfigurationValue: 300))) // every 5 min
+ 	}
+	result << response(encap(meterGet(scale: 0)))
+	result << response(encap(meterGet(scale: 2)))
+}
+
+def reset() {
+	encapSequence([
+		meterReset(),
+		meterGet(scale: 0)
+	])
+}
+
+def meterGet(scale)
+{
+    zwave.meterV2.meterGet(scale)
+}
+
+def meterReset()
+{
+    zwave.meterV2.meterReset()
+}
+
+def isNano()
+{
+    return (zwaveInfo.mfr == "0086" && zwaveInfo.prod == "0103")
+}
+
+def normalizeLevel(level)
+{
+    (int)(Math.floor((float)level * 1.01010101))
+}
+
+/*
+ * Security encapsulation support:
+ */
+def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulation cmd) {
+    def encapsulatedCommand = cmd.encapsulatedCommand(commandClassVersions)
+    if (encapsulatedCommand) {
+        log.debug "Parsed SecurityMessageEncapsulation into: ${encapsulatedCommand}"
+        zwaveEvent(encapsulatedCommand)
+    } else {
+        log.warn "Unable to extract Secure command from $cmd"
+    }
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.crc16encapv1.Crc16Encap cmd) {
+    def version = commandClassVersions[cmd.commandClass as Integer]
+    def ccObj = version ? zwave.commandClass(cmd.commandClass, version) : zwave.commandClass(cmd.commandClass)
+    def encapsulatedCommand = ccObj?.command(cmd.command)?.parse(cmd.data)
+    if (encapsulatedCommand) {
+        log.debug "Parsed Crc16Encap into: ${encapsulatedCommand}"
+        zwaveEvent(encapsulatedCommand)
+    } else {
+        log.warn "Unable to extract CRC16 command from $cmd"
+    }
+}
+
+private secEncap(physicalgraph.zwave.Command cmd) {
+    log.debug "encapsulating command using Secure Encapsulation, command: $cmd"
+    zwave.securityV1.securityMessageEncapsulation().encapsulate(cmd).format()
+}
+
+private crcEncap(physicalgraph.zwave.Command cmd) {
+    log.debug "encapsulating command using CRC16 Encapsulation, command: $cmd"
+    zwave.crc16EncapV1.crc16Encap().encapsulate(cmd).format()
+}
+
+private multiEncap(physicalgraph.zwave.Command cmd, Integer ep) {
```

Comment:
Why was multichannel added?

Reply:
Just an oversight from using existing code.  Will remove this method.